docs/sql-reference/sql-statements/auto_increment.md

```diff
@@ -10,7 +10,7 @@ When a new data row is loaded into a table, StarRocks automatically assigns a gl
 - Join tables: When multiple tables are joined, an `AUTO_INCREMENT` column can be used as the Join Key, which can expedite queries compared to using a column whose data type is STRING, for example, UUID.
 - Count the number of distinct values in a high-cardinality column: An `AUTO_INCREMENT` column can be used to represent the unique value column in a dictionary. Compared to directly counting distinct STRING values, counting distinct integer values of the `AUTO_INCREMENT` column can sometimes improve the query speed by several times or even tens of times.
 
-You need to specify an `AUTO_INCREMENT` column in the CREATE TABLE statement. The data types of an `AUTO_INCREMENT` column must be BIGINT. The value for an AUTO_INCREMENT column can be [implicitly assigned](#assign-values-implicitly) or [explicitly specified](#specify-values-explicitly). It starts from 1, and increments by 1 for each new row.
+You need to specify an `AUTO_INCREMENT` column in the CREATE TABLE statement. The data types of an `AUTO_INCREMENT` column must be BIGINT. The value for an AUTO_INCREMENT column can be [implicitly assigned or explicitly specified](#assign-values-for-auto_increment-column). It starts from 1, and increments by 1 for each new row.
```

Comment:
i think the following definition of auto_increment is more clearer, maybe we can learn from this 
![image](https://github.com/StarRocks/starrocks/assets/98087056/52aac039-f5b6-43d8-9121-04c77e7895a7)